help/user-guide/create-sample-isajson-compliant.md

```diff
@@ -79,13 +90,19 @@ Two input values:
 
 ![create study samples 6]({{ "/images/user-guide/isajson-compliance/create_samples_isastudy_samples_6.png" | relative_url }}){:.screenshot}
 
-Navigate to the same Sources table from the Study design tab, click on "Choose File" button at the bottom of the page, select the saved excel file and click "Upload". Verify and confirm the upload via the pop-up window, then click "Save".
+<div class="alert alert-info">
```

Comment:
This should really be in a `{% include callout.html type="warning" content="The syntax shown..." %}` or perhaps `type="important"` (your choice). The way you have used the line breaks and the colon is not clear to me, are you trying to make one sentence then two bullet points? I am not sure how to do multiple paragraphs within the callout but there must be a way.